massa.iml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file ?

Reply:
Sorry, it's shouldn't be included, I'm using Intellij idea as an IDE.